CMakeLists.txt

```diff
@@ -19,18 +19,9 @@ set(GENERIC_LIB_SOVERSION "1")
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -DDEBUG")
 
-################################
-# Add targets
-# By Default shared libray is being built
-# To build static libs also - Do cmake . -DBUILD_STATIC_LIBS:BOOL=ON
-# User can choose not to build shared library by using cmake -DBUILD_SHARED_LIBS:BOOL=OFF
-# To build only static libs use cmake . -DBUILD_SHARED_LIBS:BOOL=OFF -DBUILD_STATIC_LIBS:BOOL=ON
-# To build the demo binary, use cmake . -DBUILD_DEMO:BOOL=ON
-
-option(BUILD_SHARED_LIBS "build as shared library" ON)
-option(BUILD_STATIC_LIBS "build as static library" OFF)
-option(LINK_CRT_STATIC_LIBS "link CRT static library" OFF)
-option(BUILD_DEMO "build demo binary" ON)
```

Comment:
This configuration must be specified from Aseprite repository from [third_party/CMakeLists.txt](https://github.com/aseprite/aseprite/blob/main/third_party/CMakeLists.txt), see how it's done with other libraries.